test/sql/test_time_fn/R/test_time_fn

```diff
@@ -787,6 +787,26 @@ select date_diff(d0,d1,t) from test_data_diff order by t;
 10860
 -- !result
 
+-- name: test_url_encode
+select url_encode('https://docs.starrocks.io/en-us/latest/quick_start/Deploy');
+-- result:
+https%3A%2F%2Fdocs.starrocks.io%2Fen-us%2Flatest%2Fquick_start%2FDeploy
+-- !result
+select url_encode(NULL);
+-- result:
+None
+-- !result
+
+--name: test_url_decode
+select url_decode('https%3A%2F%2Fdocs.starrocks.io%2Fen-us%2Flatest%2Fquick_start%2FDeploy');
+-- result:
+https://docs.starrocks.io/en-us/latest/quick_start/Deploy
+-- !result
+select url_decode(NULL);
+-- result:
+None
+-- !result
+
```

Comment:
these cases should not be added into test_time_fn file, better to create a new file under test/sql/test_function

Reply:
ok，i move it to test_string function

Reply:
ok，i move it to string function sql testcase @silverbullet233